src/main/java/teammates/logic/core/LocalFileStorageService.java

```diff
@@ -1,28 +1,57 @@
 package teammates.logic.core;
 
+import java.io.File;
 import java.io.IOException;
-
-import javax.servlet.http.HttpServletResponse;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.nio.file.Files;
+import java.nio.file.Paths;
 
 /**
  * Holds functions for operations related to binary file storage in local dev environment.
  */
 public final class LocalFileStorageService implements FileStorageService {
 
+    private static final String BASE_DIRECTORY = "../../filestorage-dev";
+
+    private static String constructFilePath(String fileKey) {
+        return BASE_DIRECTORY + "/" + fileKey;
+    }
+
     @Override
     public void delete(String fileKey) {
-        // TODO implement this
+        File file = new File(constructFilePath(fileKey));
+        file.delete();
+    }
+
+    @Override
+    public void create(String fileKey, byte[] contentBytes, String contentType) {
+        try (OutputStream os = Files.newOutputStream(Paths.get(constructFilePath(fileKey)))) {
+            os.write(contentBytes);
+        } catch (IOException e) {
+            // TODO
+        }
     }
 
     @Override
-    public String create(String googleId, byte[] imageData, String contentType) throws IOException {
-        // TODO implement this
-        return null;
+    public boolean doesFileExist(String fileKey) {
+        try {
+            Files.newInputStream(Paths.get(constructFilePath(fileKey)));
+            return true;
+        } catch (IOException e) {
+            return false;
+        }
     }
```

Comment:
Do we need to close the `InputStream` in a `finally` block? Similarly for other functions in this file.

Reply:
The method you're commenting on has changed, but all others are in the form of `try (...) { ... }` which is [try-with-resources](https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html).